clients/js/README.md

```diff
@@ -0,0 +1,32 @@
+# deepdetect-js
+
+The official JavaScript client for interacting with the [DeepDetect](http://deepdetect.com) API.
+
+## Basic use
+
+To start, install the SDK via NPM: ```npm install deepdetect-js```
+
+
+*This will work in node.js and browsers via [Browserify](http://browserify.org/)*
+
+```js
+var DeepDetect = require('deepdetect-js');
+var app = new DeepDetect.App('http://localhost/api');
+
+```
+
+You can also use the SDK by adding this script to your HTML:
```

Comment:
This is the easiest usage, can we put it first, i.e. before the installation option via npm, and also separate the two (make two sub-section). Thanks!